packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -534,6 +499,14 @@ mixin _TapStatusTrackerMixin on OneSequenceGestureRecognizer {
   Timer? _consecutiveTapTimer;
   Offset? _lastTapOffset;
 
+  /// Callback used to indicate that a tap tracking has started.
+  VoidCallback? onTapTrackStart;
+
+  /// Callback used to indicate that a tap tracking has been reset which happens
+  /// when the timer between two taps elapses, the recognizer loses the arena,
```

Comment:
nit: `which happens on the next [PointerDownEvent] after the timer between two taps elapses`.